system/jlib/jexcept.cpp

```diff
@@ -1050,7 +1050,7 @@ void excsighandler(int signum, siginfo_t *info, void *extra)
     PROGLOG("================================================");
     PROGLOG("Signal:    %d %s",signum,strsignal(signum));
     PROGLOG("Fault IP:  %08X", ip);
-    PROGLOG("Accessing: %08X", info->si_addr);
+    PROGLOG("Accessing: %08X", (unsigned) info->si_addr);
```

Comment:
doesn't this mean you're truncating a 64-bit pointer to fit the format?
Wouldn't use of %p (or %08p, or %016p) be a better fix?